src/firefly/js/ui/dynamic/EmbeddedPositionSearchPanel.jsx

```diff
@@ -296,17 +246,17 @@ const Header = function({isOpen, doSearch, targetKey, sizeKey, polygonKey}) {
     return (
         isOpen ?
             <div/>:
-            <Stack p={0}>
+            <Stack p={0} width={1}>
                 <FormPanel
-                    onSuccess={(request) => doSearch(request)}
-                    direction={'row'}
-                    width={'100%'}
+                    onSuccess={slotProps?.formPanel.onSuccess}
+                    direction='row'
+                    sx={{width:1}}
                     slotProps={{
                         searchBar: {p:0, justifyContent: 'right'},
                     }}
                     cancelText=''>
                     <Stack {...{width:'100%', alignItems:'center'}}>
-                        <Slot component={SearchSummary} request={reqObj}/>
+                        <Slot {...{component:SearchSummary, slotProps:slotProps.header, request:reqObj}}/>
```

Comment:
If you meant `slotProps.searchSummary` instead of `slotProps.header`, then ignore my previous comment.